contracts/helper/RebalancingSetCTokenBidder.sol

```diff
@@ -264,7 +271,10 @@ contract RebalancingSetCTokenBidder is
                     );
 
                     // Mint cToken using underlying
-                    cTokenInstance.mint(underlyingQuantity);
+                    require(
```

Comment:
Instead, I'd prefer since I don't like putting actions in a require
```
uint256 mintResponse = cTokenInstance.mint(underlyingQuantity);
require(mintResponse == 0, 'blah');
```